lib/src/header_parser/utils.dart

```diff
@@ -52,7 +52,7 @@ void logTuDiagnostics(
 
 extension CXSourceRangeExt on Pointer<clang_types.CXSourceRange> {
   void dispose() {
-    free(this);
+    malloc.free(this);
```

Comment:
more malloc

Reply:
Done, replaced all `malloc` with `calloc`.
Is there any difference in using `malloc` vs `calloc` when freeing?

Reply:
No, both just use normal `free`. But it's nice to be consistent.